pkg/controller/workspace/config/config.go

```diff
@@ -47,6 +46,23 @@ const (
 	ConfigMapNamespaceEnvVar = "CONTROLLER_CONGIG_MAP_NAMESPACE"
 )
 
+const (
+	ServerImageName = "cherestapis.image.name"
+
+	SidecarPullPolicy = "sidecar.pull.policy"
+
+	PluginRegistryURL = "plugin.registry.url"
+
+	IngressGlobalDomain = "ingress.global.domain"
+
+	//WorkspacePVCName config property handles the PVC name that should be created and used for all workspaces within one kubernetes namespace
+	WorkspacePVCName             = "pvc.name"
+	WorkspacePVCStorageClassName = "pvc.storageclass.name"
+
+	UnifiedPluginBrokerImage = "che.workspace.plugin_broker.unified.image"
+	InitPluginBrokerImage    = "che.workspace.plugin_broker.init.image"
+)
+
```

Comment:
Since these seem to be accessed via `Get` methods, it may make sense to make these not external.